pkg/service/publish_logs.go

```diff
@@ -171,16 +170,12 @@ func (s *grpcServer) PublishLogs(ctx context.Context, req *pb.LogCollection) (*p
 func (mw logmw) PublishLogs(ctx context.Context, nodeKey string, logType logger.LogType, logs []string) (message, errcode string, reauth bool, err error) {
 	defer func(begin time.Time) {
 		uuid, _ := uuid.FromContext(ctx)
-		logger := level.Debug(mw.logger)
-		if err != nil {
-			logger = level.Info(mw.logger)
-		}
-		logger.Log(
+
+		mw.knapsack.Slogger().Log(ctx, levelForError(err), message,
```

Comment:
For this one and PublishResults, it looks like the message is empty (at least in the successful case) -- do we want to do `"publish logs"` and `"publish results"` for the log message instead, like how you've got it for e.g. RequestConfig?

Reply:
I'm not quite sure what is correct. Possibly using `"publish logs"` when message is empty. see previous conversation on subject https://github.com/kolide/launcher/pull/1436#discussion_r1380905299

Reply:
So these cases are a successful completion of the thing, right? And we have `method` there. So maybe just "success"?

Reply:
I'm fine with `success` or `publish logs`!

Reply:
The case on "publish logs" feels wrong. Could be "published logs" or even "published"

Consider something like scoping the view down to just showing the publish logs method.

Reply:
I've updated all the publish methods to either have success of failure as the message